packages/client/src/devices/__tests__/SpeakerManager.test.ts

```diff
@@ -0,0 +1,66 @@
+import { afterEach, beforeEach, describe, vi, it, expect } from 'vitest';
+import { mockAudioDevices } from './mocks';
+import { of } from 'rxjs';
+import { SpeakerManager } from '../SpeakerManager';
+import { checkIfAudioOutputChangeSupported } from '../devices';
+
+vi.mock('../devices.ts', () => {
+  console.log('MOCKING devices');
+  return {
+    getAudioOutputDevices: vi.fn(() => of(mockAudioDevices)),
```

Comment:
This could be a stupid doubt but, why are we using audio input devices for mocking the audio output devices? I see that the `mockAudioDevices` mostly contains the microphone or audio input devices.

Reply:
This is because for testing purposes the actual data doesn't matter, only the shape of the data. Technically we could use the same test data for the camera and mic manager as well.